### DISCUSSION TITLE
AdminUserTableComponent.vue:74 Uncaught (in promise) SyntaxError: The requested module '/node_modules/.vite/deps/laravel-vue-pagination.js?v=9c42b56e' does not provide an export named 'default' (at AdminUserTableComponent.vue:74:8)

### DISCUSSION THREAD

**Discussion by TheHandOfKing** [gilbitron/laravel-vue-pagination#162] [Help & support]
Nothing much here to explain, when trying to implement this package I get the error above

**Version information (please complete the following information):**
 - Vue: [3.0.0]
 - Laravel: [10]

**Comment by gilbitron** [comment]
How are you importing the package?

**Reply by TheHandOfKing** [reply]
I think the problem is with the composition api, I'm just using the regular import statement

**Comment by junedc** [comment]
I have encountered the same issue on a typescript import.  As an example project https://github.com/wobsoriano/v-dashboard


```
import { createApp } from 'vue';
import App from './App.vue';
import router from './router';
import './assets/main.css';

import LaravelVuePagination from 'laravel-vue-pagination';

const app = createApp(App);
app.component('Pagination', LaravelVuePagination)
app.use(router);
app.mount('#app');
```
